api/WindowEventHandlers.json

```diff
@@ -6,40 +6,40 @@
         "spec_url": "https://html.spec.whatwg.org/multipage/webappapis.html#windoweventhandlers",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "1"
           },
           "chrome_android": {
-            "version_added": true
+            "version_added": "18"
           },
           "edge": {
             "version_added": "12"
           },
           "firefox": {
-            "version_added": true
+            "version_added": "1"
           },
           "firefox_android": {
-            "version_added": true
+            "version_added": "4"
```

Comment:
Support for the `WindowEventHandlers` doesn't really mean anything, but setting this to the earliest version of any of the subfeatures seems fine until `WindowEventHandlers` is removed.

Reply:
That was my thought process, so that it doesn't impact our real values score until then!